examples/influxDB2-nifi/README.md

```diff
@@ -0,0 +1,145 @@
+--
+
+This section explains why adaptations is needed to make run InfluxDB (version 2.X) with Nifi.
+
+### Problem with existing InfluxDB connectors
+
+Nifi already embeds processors to query and insert data (e.g. 'ExecuteInfluxDBQuery').
+Currently, those processors run with a version 1.X of InfluxDB   
+Using those embedded connectors will raise authentification error if they are used with InfluxDB version 2.X.
+
+```
+influxdb.error.message
+{"code":"unauthorized","message":"Unauthorized"}
+```
+
+### Looking for solutions
+
+It exists other connectors to communicate to InfuxDB version 2.X but, at this time,
+they are not built with nifi ([see bundle Nifi](https://github.com/influxdata/nifi-influxdb-bundle#about-this-project)).
+This error is also defined on [stackoverflow](https://stackoverflow.com/questions/70706387/send-data-from-sql-server-to-influxdb-via-apache-nifis-putinfluxdb-processor)
+
+It also seems that changing the version of [Nifi](https://nifi.apache.org/docs/nifi-docs/html/getting-started.html#downloading-and-installing-nifi) (1.12.1 and 1.15.2 tested) doesn't have impact on the InfluxDB connectors.
+Only the version 1.X of influxDB is taken into account.
+
+The only solution found is adding the new processor [updating the Nifi chart](https://github.com/cetic/helm-nifi#use-custom-processors).
+After Nifi reloading, the new libraries are activated.
+
+Installing Fadi
+--
+To run the example at least Nifi, Traefik and InfluxDB must be installed.
+
+### Preparing the installation
+
+Before installing Fadi, the Fadi chart must be customized with the extra libraries of Nifi. 
+If you don't use Nifi v1.12.1 (Fadi currently uses this version), 
+[download](https://github.com/influxdata/nifi-influxdb-bundle#installation) and update the library that fits your needs 
+in the `nifi-influx-nar` folder.
+
+The file `value.yaml` will install Fadi with Nifi, Traefik and InfluxDB. It will also 
+customize the Nifi chart with the downloaded libraries.
+
+### Running the installer
+
+1. in `values.yaml`file, 
+   1. update the Nifi field `path: "/Users/<YOUR_PATH>/nifi-influx-nar"`
+   2. update the InfluxDB password 
+   ```
+      password: "<YOUR_PASSWORD>"
+      token: "<YOUR_PASSWORD>"
+   ```
+2. run `./deploy.sh`
+
+### Testing
+
+The current Nifi version (v 1.12.1) cannot be reached by a port-forwarding.
+Treafik must be used. To run Treafik, an ip must be provided.
+
+if you are using minikube on a local cluster:
+1. run `minikube tunnel` to associate an ip to your cluster (LB to Treafik)
+2. configure your `/etc/hosts` file
+```
+<IP_TUNNEL> nifi.test.local
+```
+3. connect yourself to http://nifi.test.local/nifi/ to reach nifi
+4. forward the port of influxDB 
+
+Running the use case
+--
+
+At this point your must be able to connect yourself to Nifi and InfluxDB.
+
+### Importing data into InfluxDB
+
+Data could be imported with a file or using a copy-paste function.
+
+On the lateral bar choose `Data` and create a bucket. 
```

Comment:
What lateral bar, are we in Nifi or InfluxDB web interface?

--> add a link 
--> add a screenshot